WireNetwork/Sources/WireNetwork/Network/NetworkStack/REST API/UnauthenticatedRESTAPI.swift

```diff
@@ -16,18 +16,14 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-import Foundation
+public struct UnauthenticatedRESTAPI: Sendable {
```

Comment:
Similar to `AuthenticatedRESTAPI` but doesn't need an api service, just a network service...